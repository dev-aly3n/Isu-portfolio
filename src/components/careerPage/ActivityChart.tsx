const ActivityChart: React.FC = () => {
  return (
    <div>
      <h2 className="text-5xl text-gray-300 mb-10 mx-auto text-center mt-5">
        Coding Activity
      </h2>
      <p className="w-10/12">
        This chart just showing the TIME of TYPING code. So it not considering
        the time of thinking, searching at google, watching video about code or
        even reading the docs. Maybe multiply it by 2 or 3 can show my real
        coding activity.
      </p>
      <figure className="w-full  my-10 px-5 md:w-[768px]">
        <embed src="https://wakatime.com/share/@aly3n/7db21f10-1621-4333-8f9b-0a9a49fa6a54.svg"></embed>
      </figure>
      <figure className="w-full  my-10 px-5 md:w-[768px]">
        <embed src="https://wakatime.com/share/@aly3n/7b14775a-13a0-4884-a8f3-a4f40b134860.svg"></embed>
      </figure>
    </div>
  );
};

export default ActivityChart;
