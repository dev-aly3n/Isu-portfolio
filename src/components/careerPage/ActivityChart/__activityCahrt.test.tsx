import { render, screen } from "@testing-library/react";
import React, { useRef } from "react";
import { RefObject } from "react";
import ActivityChart from ".";

const ref = new Object() as RefObject<HTMLDivElement>;


describe("career/contacts", () => {
  it("snap", () => {
    const { container } = render(<ActivityChart />);
    expect(container).toMatchSnapshot();
  });
});
