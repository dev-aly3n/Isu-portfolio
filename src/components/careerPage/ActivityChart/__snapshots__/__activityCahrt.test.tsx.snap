// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`career/contacts snap 1`] = `
<div>
  <div
    style="opacity: 0;"
  >
    <h2
      class="text-5xl text-gray-300 mb-10 mx-auto text-center mt-5"
    >
      Coding Activity
    </h2>
    <p
      class="w-10/12"
    >
      This chart just showing the TIME of TYPING code. So it not considering the time of thinking, searching at google, watching video about code or even reading the docs. Maybe multiply it by 2 or 3 can show my real coding activity.
    </p>
    <figure
      class="w-full my-10 px-5 md:w-[768px]"
    >
      <embed
        src="https://wakatime.com/share/@aly3n/28c1bd08-d655-4249-a17e-a985ff32f4d9.svg"
      />
    </figure>
    <figure
      class="w-full my-10 px-5 md:w-[768px]"
    >
      <embed
        src="https://wakatime.com/share/@aly3n/d6968611-fd97-4c0d-8ba1-1fd7e69ba510.svg"
      />
    </figure>
  </div>
</div>
`;
